### PR TITLE
Added Hotkey for Save

### PR DESCRIPTION
**Pull request by EliasMasche**
Added a new hotkey functionality for Saving maps with `Ctrl + S` and refactored the lambda expression into its own function

### CODE REVIEW

File: src/TSMapEditor/UI/TopBar/TopBarMenu.cs [hunk 1]
```diff
@@ -55,16 +55,7 @@ public override void Initialize()
...
-            {
-                if (string.IsNullOrWhiteSpace(map.LoadedINI.FileName))
-                {
-                    SaveAs();
-                    return;
-                }
-
-                map.Save();
-            });
+            fileContextMenu.AddItem("Save", () => { SaveMap(); });
```
Comment (Rampastring) [inline src/TSMapEditor/UI/TopBar/TopBarMenu.cs]: Unnecessary brackets `{` `}` in the lambda expression. See the SaveAs one below for reference.

File: src/TSMapEditor/UI/TopBar/TopBarMenu.cs [hunk 2]
```diff
@@ -166,6 +157,18 @@ public override void Initialize()
             KeyboardCommands.Instance.GenerateTerrain.Triggered += (s, e) => EnterTerrainGenerator();
             KeyboardCommands.Instance.ConfigureTerrainGenerator.Triggered += (s, e) => windowController.TerrainGeneratorConfigWindow.Open();
             KeyboardCommands.Instance.PlaceTunnel.Triggered += (s, e) => mapView.EditorState.CursorAction = placeTubeCursorAction;
+            KeyboardCommands.Instance.Save.Triggered += (s, e) => { SaveMap(); };
```
Comment (Rampastring) [inline src/TSMapEditor/UI/TopBar/TopBarMenu.cs]: Same here.

**Review by Rampastring — CHANGES_REQUESTED** [review]
Looks good aside from a couple of stylistical issues.

**Comment by EliasMasche** [comment]
Updated commit with removal of unnecessary brackets

**Review by Rampastring — APPROVED** [review]

**Pull request merged**